message_center_compassion/views/compassion_mapping_view.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_compassion_mapping_form" model="ir.ui.view">
+        <field name="name">compassion.mapping.form</field>
+        <field name="model">compassion_mapping</field>
+        <field name="arch" type="xml">
+            <form>
+                <group>
+                    <field name="name"/>
+                    <field name="fields_json_ids">
+                        <tree editable="bottom">
+                            <field name="json_name"/>
+                            <field name="field_name"/>
```

Comment:
Here you would replace by `field_id` like I suggested. You can benefit from the mapping and propose only valid fields within the associated model by adding a domain attribute (given that you add as well the `model_id` field in the form view.
```suggestion
                            <field name="field_id" domain="[('model_id', '=', model_id)]"/>
```